docs/src/modules/spring/pages/access-control.adoc

```diff
@@ -1 +1,36 @@
 = Access Control Lists (ACLs)
+
+This section describes functionality in Spring SDK related to ACLs (Access Control Lists), for information about how the ACLs are defined and control access see xref:spring:using-acls.adoc[Using ACLs]
+
+== Default ACL in project templates
+
+If no ACLs is defined at all in a Kalix service, Kalix will allow requests from both other services and the internet to all components of a Kalix service.
+
+The Maven archetype include a less permissive ACL for the entire service, to not accidentally make services available to the public internet, just like the one described in the next section.
+
+== Defining an ACL for the entire Kalix Service
+
+A default ACL for the entire Kalix Service can be defined by placing a on the Main of your service:
+
+```java
+@Acl(allow = @Acl.Matcher(service = ""))
+public class Main { ... } 
+```
+
+This is the default ACL. It allows calls from any other Kalix service deployed in the same Kalix project and from the internet.
+
+== Inspecting the principal inside a service
+
+Checking the ACLs are in general done for you by Kalix, however in some cases programmatic access to the principal of a call can be useful.
+
+Accessing the principal of a call inside a service is possible through the request metadata `Metadata.principals()`. The `Metadata` for a call is available through the context (`actionContext`, `commandContext`) of the component.
+
+== ACLs when running unit tests
+
+In the generated unit test testkits, the ACLs are ignored.
+
+== ACLs when running integration tests
+
+When running integration tests, ACLs are disabled by default but can be explicitly enabled per test by running the test with `Settings.withAclEnabled`.
```

Comment:
I just realised that the testkit for the Spring SDK won't allow us to tweak this setting.

Reply:
We will need to change the testkit to allow users to enable it and also to allow users to set the `serviceName`. So, for now, we can only state that ACLs are disabled for integration tests. 

On follow-up PR we can change the testkit and then add these sentences back.

Reply:
Maybe create a issue for this so we don't forget about it?

Reply:
https://github.com/lightbend/kalix/issues/7906